lib/sidekiq/form526_backup_submission_process/processor.rb

```diff
@@ -332,7 +332,9 @@ def get_form526_pdf
 
         form_version = submission.saved_claim.parsed_form['startedFormVersion']
         if form_version.present?
-          resp = get_form_from_external_api(headers, ApiProviderFactory::API_PROVIDER[:lighthouse], form_json.to_json)
+          transaction_id = submission.system_transaction_id
```

Comment:
I think I would want different/separate IDs to differentiate between "normal" and backup transactions potentially? 

Like if in the backup path the ID be `"BackupForm526Submission_#{id}_#{SecureRandom.uuid}"`

There may be a reason not to... but Im not sure I can think of one.

Reply:
right now, this isn't logged anywhere, so that syntax will be in the next ticket #93563